browser/static/js/main_track.js

```diff
@@ -638,8 +639,15 @@ function render_edit_image(ctx) {
   ctx.drawImage(seg_image, padding, padding, dimensions[0], dimensions[1]);
   ctx.restore();
 
+  ctx.save();
+  let region = new Path2D();
```

Comment:
since you're not reassigning `region`, you can use `const`.

```suggestion
  const region = new Path2D();
```